gammapy/data/data_store.py

```diff
@@ -711,3 +713,59 @@ def make_psf(self, position, energy=None, theta=None):
         psf_tot = EnergyDependentTablePSF(energy=energy, offset=theta, exposure=exposure,
                                           psf_value=psf_value.T)
         return psf_tot
+
+    def make_mean_edisp(self, position, e_true, e_reco, low_reco_threshold=None, high_reco_threshold=None):
+        r"""Make mean rmf for a given position and a set of observations.
+
+        Compute the mean rmf of a set of observations j at a given position
```

Comment:
Instead of repeating the definiton here I would just add a link to the ``IRFStacker`` (because that is where the stacking is actually implemented). You can add links to a function like this
``:func:`~gammapy.irf.IRFStacker.stack_edisp``